### DISCUSSION TITLE
Swedish Translation - Test build

### DISCUSSION THREAD

**Discussion by shift18** [deckerst/aves#1140] [Q&A] [answered]
Hi, I've been translating aves to Swedish.
I've been meaning to finish the translation but I've been stalling,  partly because I'd like to be able to inspect my translations in context as I found it really tricky to translate some of the strings correctly without context.

Would it be possible to have a test build built so that i might inspect the translations in context before publish the translations in the regular build?

/Swedish Translator

**Comment by deckerst** [comment, marked as answer]
Sure. Another contributor completed the translation, so I'm integrating it right now.
I added both of you as power users for the Swedish translation on Weblate, so you can both change and approve the translations.
I'll put a link to a test build here soon.

**Reply by shift18** [reply]
Ohh, I didn't notice someone else was contribution to the translation.
Thought I was suppose to receive an email when some contributes on web late,  but guess not.
No matter, it's great that someone else is helping it along.

**Comment by deckerst** [comment]
[Here](https://drive.google.com/file/d/1BMfhoaV7svEhUxK1d-a7EnbHPwvrtfV8/view?usp=sharing) it is.

**Reply by shift18** [reply]
That test build came i handy, I've now basically finished the translation of Aves into Swedish.
But I'd still like another test build so that I may do a final round of quality assurance before it's published to the main build.

Please make that test build for me whenever you got the time, Thank you!

**Reply by deckerst** [reply]
Great, thanks! I'll be home on Wednesday so I'll try and make a build then. And don't worry too much, the translation doesn't need to be perfect and we can always improve it later.

**Reply by shift18** [reply]
> Great, thanks! I'll be home on Wednesday so I'll try and make a build then. And don't worry too much, the translation doesn't need to be perfect and we can always improve it later.

Yes , I do happen to be a bit of a perfectionist so that might be easier said then done (especially when translating such a great app)
Anyway I've done my best to make the translation easily understood by being consistent and following the precedent set by the tech industry.

If something's of, then yes of course I'll do my best to fix it, though I'm confident that what we got right now is quite decent and a lot better then no translation.

**Reply by deckerst** [reply]
Here is a build: https://drive.google.com/file/d/1K1Nzt0Cp9KuiJva6EgzCFG3NzU5HLncu/view?usp=sharing

**Reply by shift18** [reply]
This will be helpful when giving the translation a final polish, thank you!

And welcome back from your trip!
